WordPressKit/Jetpack Scan/JetpackCredentials.swift

```diff
@@ -2,10 +2,10 @@ import Foundation
 
 /// A limited representation of the users Jetpack credentials
 public struct JetpackScanCredentials: Decodable {
-    public let host: String
-    public let port: Int
-    public let user: String
-    public let path: String
+    public let host: String?
+    public let port: Int?
+    public let user: String?
+    public let path: String?
     public let type: String
```

Comment:
Are you sure these others will always be present? Just wondering if we shouldn't change them to Optionals too...

Reply:
From what I can tell the `type`, `role`, `stillValid` are always present.